app/mailers/offline_cancellation_mailer.rb

```diff
@@ -0,0 +1,20 @@
+class OfflineCancellationMailer < BaseMailer
+
+  attr_reader :reservation
+
+  def send_notification(reservation)
+    @reservation = reservation
+    @instrument = reservation.product
+    mail(to: reservation.user.email, subject: subject)
+  end
+
+  private
+
+  def subject
+    text(
+      "offline_cancellation_mailer.send_notification.subject",
```

Comment:
If we didn't need the interpolation, this method wouldn't need to exist, but I thought there would be potential to put more in the subject (facility name, downtime category, etc.).